### PR TITLE
Break out behavior model function

### PR DESCRIPTION
**Pull request by dougollerenshaw**
This PR breaks `load_behavior_model_outputs` into a distinct function in data_access.loading.

Making this function available on its own facilitates loading just the model outputs if desired.

The `add_model_outputs_to_stimulus_presentations` function now uses `load_behavior_model_outputs`

### CODE REVIEW

File: visual_behavior/data_access/loading.py [hunk 1]
```diff
@@ -489,17 +489,24 @@ def get_ophys_dataset(ophys_experiment_id, include_invalid_rois=False, from_lims
...
+    id_type = from_lims.get_id_type(ophys_experiment_id)
+    if id_type != 'ophys_experiment_id':
+        warnings.warn('It looks like you passed an id of type {} instead of an ophys_experiment_id'.format(id_type))
+
+    assert id_type == 'ophys_experiment_id', "The passed ID type is {}. It must be an ophys_experiment_id".format(id_type)
+
+    if load_from_lims:
         dataset = BehaviorOphysExperiment.from_lims(int(ophys_experiment_id))
-    elif from_nwb:
+    elif load_from_nwb:
```
Comment (yavorska-iryna) [inline visual_behavior/data_access/loading.py:509]: wondering if it's worth adding a descriptive error message here if the experiment is not found in lims or nwb; just a suggestion.

**Review by yavorska-iryna — APPROVED** [review]

**Review by alexpiet — APPROVED** [review]
This looks good, but note that this PR also merges the stuff in the other PR about session IDS, so only merge them when both are finalized

File: visual_behavior/data_access/loading.py (hunk 1 above)
Comment (dougollerenshaw) [reply visual_behavior/data_access/loading.py:509]: There is an existing statement that will print a line if there is no NWB file found:
https://github.com/AllenInstitute/visual_behavior_analysis/blob/7f04506065fc9767b41b248cdfa300f9409786bf/visual_behavior/data_access/loading.py#L511

The changes in PR #760 will cause an assertion error to be thrown if there is no associated experiment ID in LIMS.

Are there any other specific checks you're thinking of?

**Comment by dougollerenshaw** [comment]
@alexpiet after merging #760 into master, then master into this PR, the changes in this PR appear much less expansive. Only 1 file is now changed.

**Pull request merged**